url/src/comp.php

```diff
@@ -400,21 +557,19 @@ function get_item($shortCode, $itemID, $TalisGUID, $token) {
 }
 
 function check_web_addresses($oldURL, $newURL, $web_address_array, $mode) {
-	global $myfile;
 	$oldURL_found = array_search($oldURL, $web_address_array);
-	
-	if (isset($oldURL_found)) {
+	echo_message_to_screen(DEBUG, "Array Search Result = {$oldURL_found}");
+
+	if (isset($oldURL_found) && $oldURL_found > -1) {
```

Comment:
I found when I was testing that the deletion wasn't triggering properly as `array_search()` always returns a value and so `isset($oldURL_found)` always returned true. Here we check to see if it is set AND see if it has a value of zero or more.

This meant it ALWAYS deleted the web addresses if there were ANY web addresses!!

Reply:
The Php docs say it should ``false`` if there's no match, so wondering if it might be safer to assert ``!== false``.  Not tested myself...

https://www.php.net/manual/en/function.array-search.php

``false == 0`` apparently.  https://www.php.net/manual/en/types.comparisons.php
Does thata mean ``>`` will match false, or is that only for loose eqaulity ``==``? PHP be craaaaazy. 😄

Reply:
https://wtools.io/php-sandbox/b1PA

Sketching out some values in that link suggests we should be fine with detecting whether an index is greater that zero. but the `isset` test doesn't actually add anything here.